swagger-backend.yaml

```diff
@@ -239,3 +240,37 @@ paths:
           description: Error
           schema:
             $ref: '#/definitions/error'
+  '/namespaces/{namespace}/playbookexecution/{device-id}/playbookexecutions':
```

Comment:
What is the motivation to define it as a top-level collection?
Do you expect a need to list all of the playbookexecution?
I'd imagine this as a sub-collection of a specific device.
Instead, I'd expect it to look as `'/namespaces/{namespace}/devices/{device-id}/playbookexecutions'`

Reply:
Please see https://github.com/project-flotta/flotta-operator/pull/291